### DISCUSSION TITLE
some code ideas I forgot about

### DISCUSSION THREAD

**Discussion by joshualerickson** [joshualerickson/exploreRGEE#16] [Ideas]
Hey @zackarno, totally forgot about some of these functions that I converted from JS to R from a class I took last year. They might have some stuff we could implement. I'll keep looking but feel free to take a look as well. 

https://github.com/joshualerickson/gee_training_ex

**Comment by zackarno** [comment]
very cool to see some these types of scripts written out in R as opposed to JavaScript! I will have a closer look, thanks for sharing
